en_US/getting-started/install-on-k8s.md

```diff
@@ -1,116 +1,152 @@
-# Install EMQX Cluster with EMQX Operator
+# Use EMQX Operator to deploy EMQX cluster
 
-For more information about latest EMQX Operator, please access [github](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/getting-started/getting-started.md).
+The corresponding relationship between EMQX Operator and the applicable EMQX version is as follows:
 
-1. Use [cert-manager](https://github.com/cert-manager/cert-manager) for provisioning the certificates for the webhook server. Please follow the [cert-manager documentation](https://cert-manager.io/docs/installation/) to install it。
+|         EMQX Version   |                  EMQX Operator Version        |  APIVersion |
+|:----------------------:|:---------------------------------------------:|:-----------:|
+|   4.4.6<= EMQX < 4.4.8 |   1.2.5                                       |  v1beta3    |
+|   4.4.8 <= EMQX        | 1.2.6，1.2.7，1.2.8，2.0.0，2.0.1，2.0.2（推荐） |  v1beta3    |
 
-2. Install Operator Controller Manager service in one of the following ways：
 
-    + Install with default static files
+For the latest information about EMQX Operator, please visit [EMQX Operator](https://github.com/emqx/emqx-operator/blob/main/docs/zh_US/getting-started/getting-started.md). **Note**: EMQX Operator requires Kubernetes v1.20.11 or above.
```

Comment:
```suggestion
For the latest information about EMQX Operator, please visit [EMQX Operator](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/getting-started/getting-started.md). **Note**: EMQX Operator requires Kubernetes v1.20.11 or above.
```

Reply:
Thank you for your reminder, I will modify it